chrome_extension/background.js

```diff
@@ -1,10 +1,20 @@
 let lastCall = 0;
 let USER_ACCESS, USER_SECRET;
+let saved_topic;
 window.requestDone = false;
 
 chrome.runtime.onMessage.addListener(function (message, sender, callback) {
-    if (message === 'enable_page_action') {
+    if (message === 'enable_page_action' && sender.tab.active) {
         let tid = sender.tab.id;
+
+        chrome.storage.local.get(['user_logged_in'], function(results) {
```

Comment:
Does chrome.storage sync with Google's cloud? (It can on Chrome OS) Could that impact GDPR? (seeing as this might go public)